Objects/memoryobject.c

```diff
@@ -177,6 +177,7 @@ PyTypeObject _PyManagedBuffer_Type = {
 };
 
 
+
```

Comment:
I think three blank lines is too much. :)

Reply:
Absolutely! Missed that one.